controllers/users.js

```diff
@@ -37,12 +37,13 @@ class Users {
         status: 201,
         message: 'user created',
         user: {
+          id: user.id,
           email: user.email,
           username: user.username
         }
       });
     } catch (e) {
-      return e;
+      return res.send({ error: e });
```

Comment:
Most of the times the error here is an internal server error, so you should set the **status** to 500 and add a message describing the error like **internal server error.**

Reply:
Thank you @avpaul

Reply:
solved and thank you @avpaul